libs/clients/vmst/src/lib/utils.ts

```diff
@@ -8,39 +10,52 @@ export const createWrappedFetchWithLogging = (
   return new Promise((resolve, reject) => {
     fetch(input, init)
       .then(async (response) => {
-        const body = init?.body ? JSON.parse(init?.body as string) : {}
+        let requestBody = init?.body ? JSON.parse(init?.body as string) : {}
 
-        const newInit = {
-          ...init,
-          body: {
-            ...body,
-            applicant: 'hidden',
-            otherParentId: '' ?? 'hidden',
-            email: 'hidden',
-            phoneNumber: 'hidden',
-            paymentInfo: {
-              ...body?.paymentInfo,
-              bankAccount: 'hidden',
-            },
-            attachments: body?.attachments?.length,
+        // Filter known sensitive data
+        // TODO: Should pick what we need instead
+        requestBody = omit(requestBody, [
+          'applicant',
+          'otherParentId',
+          'email',
+          'phoneNumber',
+          'paymentInfo.bankAccount',
+          'employers.email',
+          'employers.approverNationalRegistryId',
+        ])
+
+        const vmstMetadata = {
+          request: {
+            body: requestBody,
```

Comment:
Is it possible to either `omit` attachments or change so we only see the number of attachments or at least don't see `attachmentBytes` only show `attachmentType`? See [DD logs](https://app.datadoghq.eu/logs?query=%40dd.service%3Aapplication-system-api%20kube_namespace%3Afeature-chorelogging-cleanup%20&agg_m=count&agg_m_source=base&agg_q=status%2Cservice&agg_q_source=base%2Cbase&agg_t=count&cols=host%2Cservice&event=AgAAAZBUWLcq_NEVJwAAAAAAAAAYAAAAAEFaQlVXTVlUQUFBNVI2VHBPa1Y5UkFBRgAAACQAAAAAMDE5MDU0NTktYzVmYi00NjgyLTk3YzEtNzE5ODRmZDJjY2Fh&fromUser=true&messageDisplay=inline&refresh_mode=sliding&sort=time&sort_m=%2C&sort_m_source=%2C&sort_t=%2C&storage=hot&stream_sort=desc&top_n=10%2C10&top_o=top%2Ctop&view=spans&viz=stream&x_missing=true%2Ctrue&from_ts=1719401176158&to_ts=1719402076158&live=true)